source/beamline/SGM/SGMEnergyCoordinator.cpp

```diff
@@ -556,7 +559,8 @@ void SGMEnergyCoordinator::onEnergySetpointPVChanged(double value)
 	if(energySetpointInitialized_)  {
 
 		qDebug() << QString("Forwarding energy setpoint value of %1 to energy control").arg(value);
-		energyControlCoordinator_->move(value);
+		AMControl* controlVersion = energyControlCoordinator_;
```

Comment:
That makes sense with with the setpoint initialization, but you are also setting the control here for no reason.